docker-compose.yml

```diff
@@ -13,8 +13,8 @@ services:
     flowdb-testdata:
         container_name: flowdb_testdata
         image: flowminder/flowdb:testdata-master 
-        ports:
-            - 9000:5432
+        expose:
```

Comment:
Do we need the expose? From https://docs.docker.com/compose/compose-file/#expose it sounds like this is for use with `links`, but not needed when using `networks`?

Reply:
Ah, maybe not. I'll try without.